apps/docs/src/guide/creating-a-fuel-dapp/index.md

```diff
@@ -185,8 +185,11 @@ Whenever you want to add a new feature to your dApp and quickly prototype things
 
 - Now that you have a basic counter dApp running and have the `npm create fuels` workflow powering you, you can start building more complex dApps using the Fuel Stack. A good place to start for ideas and reference code is the [Sway Applications Repo](https://github.com/FuelLabs/sway-applications).
 
+- As you may have noticed, there are different types of programs in your dApp, feel free to explore [Predicates](https://docs.fuel.network/docs/fuels-ts/predicates/) and [Scripts](https://docs.fuel.network/docs/fuels-ts/scripts/), which are both important differentiators in the Fuel Stack.
```

Comment:
Maybe I misinterpreted your suggestion, perhaps you should outline it in a separate issue.

Reply:
To get a consensus, do we think this is enough of an extension, or should we be giving specific code examples for extending the demo with scripts/predicates?
> cc @FuelLabs/sdk-ts

Reply:
I'm not sure what that lite version would entail @petertonysmith94 as there isn't anything analogous existing in documentation and _create fuels_ already offers predicate and scripts by default. I think it may need to have a a separate discussion / issue (rather than doing it in this thread) to come to consensus on how we would like to extend the docs.

Reply:
A picture is worth a thousand words ✍🏾  🌹  thanks for clarifying.

Reply:
3ab9c17eb3ef9d52347313981422a6d12e63f2fa